### ISSUE TITLE
setXValuesInterval is working only with numeric X Values

### ISSUE THREAD

**Issue by domekuf** [elliottb/phpgraphlib#34]
If we have strings or other on X Axis, this setting doesn't work

**Comment by rafwell** [comment]
+1, i've needed put a number before and adjust their height for crop the helper.

**Comment by bbb0** [comment]
I am collecting data every 5mins and my X axis is the time (HH:mm format) of the collected datapoint. If I set the interval to 6 (write out every half hour), it shows 00:05, 00:10, ... 06:00, 06:05, ..., not what I expect.

The solution is to define a new variable `$postition=0;` at line 379, replace `$key % $this` with `$position % $this` at the line 476, and finally increment the value of it `$position++;` at the line 511. If I can, I will correct it here as well.

**Comment by domekuf** [comment]
Yes, it was the solution I  proposed, hope it will be approved and merged ;)

**Comment by luigi37** [comment]
Thanks for this! it's a must "pull" :-) could we have it working also for HORIZONTAL axis labels? 
With 
`$graph->setXValuesHorizontal(true);`
Labels get squeezed all together to the left
Thanks
